spec/models/comment_spec.rb

```diff
@@ -1,14 +1,17 @@
 require 'rails_helper'
 
 RSpec.describe Comment, type: :model do
-  it 'posts comments' do
-    author = User.new(name: 'Tom', photo: 'Cat.png', bio: 'bio', posts_counter: 0)
-    post = Post.new(title: 'New post', text: 'Good evening', user: author, likes_counter: 0, comments_counter: 0)
-    post.save!
+  describe 'Comment model' do
+    user = User.create(name: 'Orcun', photo: 'Cat.png', bio: 'Test Bio')
+    post = Post.new(user:, title: 'test post', text: 'Testing post')
+    comment = Comment.new(user:, post:, text: 'Test Comment')
+    comment.update_comments_counter
 
-    comment_creator = User.new(name: 'Jerry', photo: 'Cat.png', bio: 'bio', posts_counter: 0)
-    post.comments.create!(text: 'This is my first post', user: comment_creator)
-    post.comments.create!(text: 'This is my second post', user: comment_creator)
-    expect(post.comments.length).to eql(2)
+    it 'Check Comments Counter' do
+      current = post.comments_counter
+      comment_new = Comment.new(user:, post:, text: 'New Test Comment')
+      comment_new.update_comments_counter
+      expect(post.comments_counter).to eq(current + 1)
+    end
```

Comment:
- [ ] None of your tests are passing successfully, this is due to the line 6 and 12 

(`post = Post.new(user:, title: 'test post', text: 'Testing post')`) 

 (`comment_new = Comment.new(user:, post:, text: 'New Test Comment')`)

![image](https://user-images.githubusercontent.com/55635977/157856757-02a4ec72-fe03-4dff-9271-21c15ccc6898.png)

![image](https://user-images.githubusercontent.com/55635977/157856813-626403f9-00a2-4069-8bd0-72ab1cc8622d.png)

Reply:
Hi @Emmyn5600, thanks for your reviews. The problem is I don't have any error on my side :smile:. I fixed the lines that you pointed. Hope it works. Have a nice day :100: